docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -352,15 +252,13 @@ Querying is provided with the field access operator which has the following synt
 `mappingColumn` must be the top-level column inside a mapping that has a UDT as its type,
```

Comment:
I would add a note pointing out that parentheses are required. This is very surprising quirk of our syntax

Reply:
fef8c8c2